pages/tarieven.vue

```diff
@@ -40,46 +40,46 @@
             </li>
           </ul>
           <p>
-            Pas op: aan de tarieven kunnen geen rechten ontleend worden, voordat we aan de slag gaan maken we definitieve afspraken.
+            Pas op: aan de tarieven kunnen geen rechten ontleend worden, voordat we aan de slag gaan, maken we definitieve afspraken.
           </p>
         </details>
         <div class="page-content__spacer"></div>
         <h2 class="heading-page">
           Appje Eitje Werkwijze
         </h2>
         <h3>
-          Geen factuurstress
+          Aan de slag als alles duidelijk is
         </h3>
         <p>
-          Geen verrassingen meer.
-          We werken op nacalculatie.
-          Gemaakte uren kun je volgen per maand, via onze tools.
-          Aan het einde van de maand corrigeren we dit en factureren na akkoord.
+          Wij helpen je om het (ontwikkel)werk in zinvolle delen op te splitsen en zorgen zo samen voor een backlog.
+          Als alles duidelijk is, kunnen we actief aan de slag!
+          Periodiek plannen we een stand-up en/of refinement om zowel de backlog als het Scrum board bij te werken.
+          Zo kunnen we elke sprint iets opleveren met behulp van CI / CD tooling.
         </p>
         <h3>
-          Altijd de laatste status
+          Altijd op de hoogte
         </h3>
         <p>
-          Via een heldere backlog i.c.m. actief board weet je altijd direct wat de status is.
-          Zo hoef je nooit meer te bellen drie dagen voor een deadline, maar het mag natuurlijk wel ;-)
+          Via het gezamenlijke Scrum board en backlog weet je altijd direct wat de status is van de werkzaamheden of features.
```

Comment:
Scrum is niet onze werkwijze, tenzij de klant of het project van de klant zo werkt
```suggestion
          Via het backlog en board weet je altijd direct wat de status is van de werkzaamheden of features.
```